src/perform_step/low_order.jl

```diff
@@ -229,3 +229,124 @@ end
 
   end
 end
+
+@muladd function perform_step!(integrator,cache::RKMil_GeneralConstantCache,f=integrator.f)
+  @unpack t,dt,uprev,u,W,p = integrator
+  dW = W.dW
+
+  if integrator.alg.is_commutative
+    WikJ = 1//2 .* vec(dW) .* vec(dW)'
+  else
+    WikJ = get_iterated_I!(integrator, cache)
```

Comment:
We should probably have a struct for WikJDiagonal, WikJCommute, WikJGeneral that dispatches to different ways of filling it? Then it could have the appropriate cache size for each case, and the cache variables won't necessarily need to be in the method cache (allowing for simultaneous different WikJ's in future methods)